### PR TITLE
Merge 1.7 to 2.0

### PR DESCRIPTION
**Pull request by grena**

### CODE REVIEW

File: CHANGELOG-1.7.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# 1.7.X (2018-01-11)
+# 1.7.17 (2018-02-01) (2018-01-11)
```
Comment (damien-carcel) [inline CHANGELOG-1.7.md]: nope, one is enough :-)

Comment (willy-ahva) [inline CHANGELOG-1.7.md]: remove (2018-01-11)

File: docker-compose.yml.dist [hunk 2]
```diff
@@ -81,7 +80,7 @@ services:
       MYSQL_PASSWORD: akeneo_pim
       MYSQL_DATABASE: akeneo_pim
     ports:
-      - '3306:3306'
+      - '33006:3306'
```
Comment (willy-ahva) [inline docker-compose.yml.dist]: seams weird

File: docker-compose.yml.dist [hunk 3]
```diff
@@ -93,7 +92,7 @@ services:
       MYSQL_PASSWORD: akeneo_pim
       MYSQL_DATABASE: akeneo_pim
     ports:
-      - '3307:3306'
+      - '33007:3306'
```
Comment (willy-ahva) [inline docker-compose.yml.dist]: this one too

File: docker-compose.yml.dist (hunk 2 above)
Comment (damien-carcel) [reply docker-compose.yml.dist]: Nope, it is a wanted change

**Pull request merged**